src/experiment/experiment.js

```diff
@@ -1,23 +1,24 @@
 // npm run serve:dev
-import { mediaAssets, preloadTrials } from './loadassets';
 import { initJsPsych } from 'jspsych';
 import surveyText from '@jspsych/plugin-survey-text';
 import 'jspsych/css/jspsych.css';
 import 'regenerator-runtime/runtime';
 import jsPsychRdk from '@jspsych-contrib/plugin-rdk';
 import { pressKey } from '@jspsych/test-utils';
-
-import { rootDoc } from '../firebaseConfig';
 import jsPsychPavlovia from '../jsPsychPavlovia';
 import '../css/custom.css';
 
+import { generateAssetObject, createPreloadTrials } from "@bdelab/roar-utils";
+import assets from '../../assets.json'
+import 'regenerator-runtime/runtime';
+
 // Audio files
 import jsPsychAudioKeyboardResponse from '@jspsych/plugin-audio-keyboard-response';
 
 // ---------Prepare the main timeline---------
 import { enterFullscreen } from './trials/fullScreen';
-import { introductionTrials, ifKeyboardInstrutions, ifButtonInstrutions, intro5 } from './trials/introduction';
-import { IBI1, IBI2, IBI3, IBI4, IBI5, IBIEnd } from './trials/ibi';
+import { welcome, introTrialsMapped, ifKeyboardInstrutions, ifButtonInstrutions, intro5 } from './trials/introduction';
+import { trialsMapped, IBIEnd } from './trials/ibi';
 
 // Copied camelCase from preload.js
 export const camelCase = (inString) =>
```

Comment:
We can get rid of this since we're not using it anywhere. Along with the preloadObj2contentObj function.